src/App.tsx

```diff
@@ -1,24 +1,299 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { addTodo, deleteTodo, getTodos } from './api/todos';
 
-const USER_ID = 0;
+const USER_ID = 11361;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [updatedTodos, setUptatedTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState<string>('');
+  const [sortBy, setSortBy] = useState<string>('all');
+  const [isTodoDeleted, setIsTodoDeleted] = useState(false);
+  const [todosNotCompleted, setTodosNotCompleted] = useState(0);
+  const [isLoading, setIsLoading] = useState(false);
+  const [selectedTodo, setSelectedTodo] = useState<number[]>([]);
+  const [inputValue, setInputValue] = useState<string>('');
+  const [isLoadingTodo, setIsLoadingTodo] = useState(false);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+
+  useEffect(() => {
```

Comment:
You could export this logic to a separate hook, for example you could create `useGetTodos` hook where you return an object holding your loading state and data 
<img width="482" alt="image" src="https://github.com/mate-academy/react_todo-app-add-and-delete/assets/87986955/172f8f06-3fa3-4eed-bb31-647bfa49c3bc">